### PR TITLE
Add sidebar

### PR DESCRIPTION
**Pull request by ganewman**
Add a sidebar (with styling) as well as basic placeholder pages for sidebar links.

### CODE REVIEW

File: portfolio/src/main/webapp/about.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/about.html:4]: Put this line right below the DOCTYPE

File: portfolio/src/main/webapp/gallery.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/gallery.html:4]: Put this right below DOCTYPE

File: portfolio/src/main/webapp/hobbies.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/hobbies.html:4]: Put this right below DOCTYPE

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -8,14 +8,36 @@ body {
...
 }
 
+#content h1 {
+  text-align: center;
+  color: #766;
+}
+
 #selfie {
   display: block;
+  height: 500px;
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/style.css:18]: What's the reason for changing this order?

Comment (ganewman) [reply portfolio/src/main/webapp/style.css:18]: The style guide says that declarations should be alphabetized.

File: portfolio/src/main/webapp/about.html (hunk 1 above)
Comment (ganewman) [reply portfolio/src/main/webapp/about.html:4]: Sorry, I'm not sure I understand. Which line?

Comment (suparnabarua) [reply portfolio/src/main/webapp/about.html:4]: <!DOCTYPE html> 
<meta charset="UTF-8">
Did the review comment not highlight which line this comment referred to? This applies to all the other comments below

Comment (kellyktan) [reply portfolio/src/main/webapp/about.html:4]: Also, I think the "utf-8" should be lowercased: https://google.github.io/styleguide/htmlcssguide.html#Encoding

Comment (ganewman) [reply portfolio/src/main/webapp/about.html:4]: It looks like 3-4 lines have been highlighted. I'm a little confused about what I should do.

Comment (suparnabarua) [reply portfolio/src/main/webapp/about.html:4]: This was the formatting I referred to 
<!DOCTYPE html>
<meta charset="utf-8">
Thanks Kelly for the capitalization pointer

File: portfolio/src/main/webapp/about.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```
Comment (suparnabarua) [inline portfolio/src/main/webapp/about.html:1]:
```suggestion
<!DOCTYPE html>
<meta charset="UTF-8">
```

Comment (suparnabarua) [reply portfolio/src/main/webapp/about.html:1]: Apply this to the other places as well

Comment (ganewman) [reply portfolio/src/main/webapp/about.html:1]: Fixed on all files!

**Pull request merged**